scaper/core.py

```diff
@@ -1615,6 +1640,20 @@ def _generate_audio(self, audio_path, ann, reverb=None,
             (no reverberation) and 1 (maximum reverberation). Use None
             (default) to prevent the soundscape from going through the reverb
             module at all.
+        save_isolated_events : bool
+            If True, this will save the isolated event audio in a directory adjacent to the soundscape
+            mixture, or to the path defined by `isolated_events_path`. The audio of the isolated events sum 
+            up to the mixture if reverb is not applied. Isolated events can be found 
+            (by default) at `<audio_outfile parent folder>/<audio_outfile name>_events`.
+            Isolated event file names follow the pattern: `[role][count]_[label]`, where count 
+            is the index of the isolated event in 
+            self.fg_spec or self.bg_spec (this allows events of the same label to be added more than 
+            once to the soundscape without breaking things). Role is "background" or "foreground".
+            For example: `foreground0_siren.wav` or `background0_park.wav`.
+        isolated_events_path : str
+            Like `audio_path`, this determines the path of the directory within which 
+            the isolated event audio will be saved. If None, defaults to the parent 
+            folder of audio_path.
```

Comment:
I believe this should just say:

"Path to folder for saving isolated events. If None, defaults to the parent folder of audio_path"